python/bindings.cpp

```diff
@@ -443,5 +501,51 @@ PYBIND11_MODULE(libsonata, m) {
 
     bindStorageClass<EdgeStorage>(m, "EdgeStorage", "EdgePopulation");
 
+    py::class_<SpikeReader::Population>(m, "SpikePopulation", "A population inside a SpikeReader")
+        .def(
+            "get",
+            [](const SpikeReader::Population& self) { return self.get(); },
+            "Return all spikes")
+        .def(
+            "get",
+            [](const SpikeReader::Population& self, double tstart, double tstop) {
+                return self.get(Selection({}), tstart, tstop);
+            },
+            "Return spikes between 'tstart' and 'tstop'",
+            "tstart"_a,
+            "tstop"_a)
+        .def(
+            "get",
+            [](const SpikeReader::Population& self, Selection sel) { return self.get(sel); },
+            "Return spikes with all those node_ids",
+            "node_ids"_a)
+        .def("get",
+             &SpikeReader::Population::get,
+             "Return spikes with all those node_ids between 'tstart' and 'tstop'",
+             "node_ids"_a,
+             "tstart"_a,
+             "tstop"_a)
+        .def_property_readonly(
+            "sorting",
+            [](const SpikeReader::Population& self) {
+                auto s = self.getSorting();
+                if (s == SpikeReader::Population::Sorting::none)
+                    return "none";
+                if (s == SpikeReader::Population::Sorting::by_id)
+                    return "by_id";
+                if (s == SpikeReader::Population::Sorting::by_time)
+                    return "by_time";
```

Comment:
I just checked and GCC doesn't throw any warning while Clang does.
We could just return "none" at the end since by default no sorting can be assumed.